Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLink.cs

```diff
@@ -310,6 +333,23 @@ void Start()
                 // Set master name once on start
                 FindAndUpdateMasterName();
             }
+#elif UNITY_WEBGL && !UNITY_EDITOR
+
+            SetupAnalyserSpace();
+            audioLinkWebPeer = new WebALPeer();
+
+            WebALID = UnityEngine.Random.Range(0, 99999);
+
+            LinkAnalyser(WebALID, audioSource.clip.length, 4096);
+
+            Application.focusChanged += (focus) => {
```

Comment:
weird formatting?

Reply:
how so?

Reply:
egyptian bracing. The rest of the file doesn't use it

```
Application.focusChanged += (focus) => 
{
    if (_audioLinkEnabled) 
    {
        if (focus) 
        {
        LinkAnalyser(WebALID, audioSource.clip.length, 4096);
        } 
    else 
        UnlinkAnalyser(WebALID);
    }
                
                ```